src/libav_streamer.cpp

```diff
@@ -325,36 +253,23 @@ void LibavStreamer::sendImage(const cv::Mat &img, const rclcpp::Time &time)
 
     double seconds = (time - first_image_timestamp_).seconds();
     // Encode video at 1/0.95 to minimize delay
-    pkt.pts = (int64_t)(seconds / av_q2d(video_stream_->time_base) * 0.95);
-    if (pkt.pts <= 0)
-      pkt.pts = 1;
-    pkt.dts = AV_NOPTS_VALUE;
+    pkt->pts = (int64_t)(seconds / av_q2d(video_stream_->time_base) * 0.95);
```

Comment:
This magic number of 0.95 would be nice to have as an actual constant defined in an anonymous namespace at the top of this file.

Reply:
I think this is out of scope for this PR. Instead of making it a constant I might add URL-controller parameter for this (e.g. `playback_speed`) in the future.